### DISCUSSION TITLE
How to use with plain html(/php) and css files?

### DISCUSSION THREAD

**Discussion by Pkhunterk** [FullHuman/purgecss#897] [General]
I’m using none of the gulp or grunt type tools. It’s a simple Wordpress website and I’m using UIKit framework to create a theme. It’s clear I haven’t used all the CSS supplied with UIKit. Can I use purgecss by adding the various generated html pages of the site and the css files?

**Comment by WebMechanic** [comment]
well, PurgeCSS requires Node to work (or something like it) but you don't need a task runner or bundler to use it.

You have to install Node on your machine then add PurgeCSS as a global command line tool using `npm`. 
`npm install purgecss --global`.

Then call the `purgecss` command from inside your terminal and feed it the final .css and (static) .html files that would go into your CMS theme.
In your folder with the static html and css files better add a `purge.config.js` to safe you from typing lengthy command lines.

Hope this helps.
